src/visitors/pages/__tests__/Signup.test.js

```diff
@@ -62,30 +65,60 @@ describe('Visitor Registration Component', () => {
     ] = await waitForElement(() => [
       tools.getByLabelText('Full Name'),
       tools.getByLabelText('Email Address'),
-      tools.getByLabelText('Phone Number (optional)'),
+      tools.getByLabelText('Phone Number'),
       tools.getByLabelText('Gender'),
       tools.getByLabelText('Year of Birth'),
       tools.getByTestId('emailConsent'),
       tools.getByText('CONTINUE'),
     ]);
 
-    fullName.value = visitor.name;
-    email.value = visitor.email;
-    phoneNumber.value = visitor.phoneNumber;
-    gender.value = visitor.gender;
-    birthYear.value = visitor.birthYear;
-    emailConsent.value = visitor.emailConsent;
-
-    fireEvent.change(fullName);
-    fireEvent.change(email);
-    fireEvent.change(phoneNumber);
-    fireEvent.change(gender);
-    fireEvent.change(birthYear);
-    fireEvent.change(emailConsent);
+    fireEvent.change(fullname, { target: { value: visitor.name } });
+    fireEvent.change(email, { target: { value: visitor.email } });
+    fireEvent.change(phoneNumber, { target: { value: visitor.phoneNumber } });
+    fireEvent.change(gender, { target: { value: visitor.gender } });
+    fireEvent.change(birthYear, { target: { value: visitor.birthYear } });
+    fireEvent.change(emailConsent, { target: { value: visitor.emailConsent } });
     fireEvent.click(submit);
 
     await wait(() => {
       expect(tools.history.location.pathname).toBe('/visitor/signup/thankyou');
     });
   });
+
+  test('submit form w/o email or phone number', async () => {
```

Comment:
Shouldn't we also have a test for successful submission with either email or phone number?

Reply:
Done